ping-meta/action.yml

```diff
@@ -0,0 +1,19 @@
+name: ping-meta
+description: "Pings the robotpy-meta repository"
+inputs:
+  name:
```

Comment:
This needs to be read from pyproject.toml, otherwise dist.yml will differ slightly for each project. At the moment it can be copy/pasted as long as the projects support all OS.

Reply:
Ok. Can I pip install toml in this and assume that python will be setup? Or, should I parse in bash?

Reply:
You could just create a JavaScript action.

Reply:
A javascript action would probably be the way to go here.